sklearn/linear_model/logistic.py

```diff
@@ -1045,7 +1045,9 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         Algorithm to use in the optimization problem.
 
         - For small datasets, 'liblinear' is a good choice, whereas 'sag' and
-            'saga' are faster for large ones.
+            'saga' are faster for large ones. 'liblinear' doesn't work with
```

Comment:
I am not sure that I mention anything here. IMO, in `n_jobs` this is enough.
Let see, what is the opinion of another reviewer.